src/components/MultiTagSelection.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import Downshift from 'downshift'
+
+class MultiDownshift extends React.Component {
```

Comment:
Why you name it differently from your file name?

Reply:
Is there any reason that only implementing this feature as class-base component?

Reply:
Whole component lacks of semicolon. Please update it.

Reply:
comment added in file :

// This component is for building tags-selecting-function; this function is moved to s2
// Belows are sample cods of liberary MultiDownshift